src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs

```diff
@@ -152,6 +152,28 @@ public async Task<bool> EpaOrganisationStandardExists(string organisationId, int
             }
         }
 
+        public async Task<bool> EpaOrganisationAlreadyUsingName(string organisationName, string organisationIdToExclude)
+        {
+         
+            using (var connection = new SqlConnection(_configuration.SqlConnectionString))
+            {
+                if (connection.State != ConnectionState.Open)
+                    await connection.OpenAsync();
+                
+                var sqlToCheckExists =
+                    "select CASE count(0) WHEN 0 THEN 0 else 1 end result FROM [Organisations] " +
+                    $@"WHERE EndPointAssessorName = '{organisationName}'";
+                
+                if (!string.IsNullOrEmpty(organisationIdToExclude))
+                {
+                    sqlToCheckExists =  "select CASE count(0) WHEN 0 THEN 0 else 1 end result FROM [Organisations] " +
+                                        $@"WHERE EndPointAssessorName = '{organisationName}' AND  EndPointAssessorOrganisationId != '{organisationIdToExclude}'";
```

Comment:
Can you parameterise these string interpolations?  I'm not sure whether Sql Injection can happen with `{organisationName}` but better safe than sorry I guess. 😀